sqlglot/dialects/clickhouse.py

```diff
@@ -611,6 +612,21 @@ def _parse_alter_table_replace(self) -> t.Optional[exp.Expression]:
                 exp.ReplacePartition, expression=partition, source=self._parse_table_parts()
             )
 
+        def _parse_projection_def(self) -> t.Optional[exp.ProjectionDef]:
+            if not self._match(TokenType.PROJECTION):
+                return None
+
+            return self.expression(
+                exp.ProjectionDef,
+                this=self._parse_id_var(),
+                expression=self._parse_wrapped(self._parse_statement),
```

Comment:
Why is `_parse_wrapped` required here? Can't we just use `_parse_statement()` directly and get rid of `self.wrap(...)` in the generator method?

Reply:
I've used this to require left and right parens wrapping the query, is there a way to do this one better?

Reply:
I guess this is fine. FWIW the difference is that this parses it as a `Select` whereas a single `self._parse_statement` will parse it as a `Subquery`.